plopfile.js

```diff
@@ -1,7 +1,10 @@
 const pascalCase = require('pascal-case');
+const { snakeCase } = require('snake-case');
 
 module.exports = (plop) => {
-  plop.setHelper('upperCase', (txt) => txt.toUpperCase());
+  plop.setHelper('spacedUpperCase', (txt) =>
+    snakeCase(txt).split('_').join(' ').toUpperCase(),
```

Comment:
I used `snakeCase` so that it would correctly handle pascal cased component names, which would otherwise require me to go through and check each character to see if it was a lowercase letter or not.